### DISCUSSION TITLE
Intellisense for default theme tokens is not working in VS Code

### DISCUSSION THREAD

**Discussion by ramesh-km** [chakra-ui/chakra-ui#5151] [Q&A]
Autocomplete not working for some component props. Eg: color prop on Heading component have type `any`

https://codesandbox.io/s/ecstatic-rgb-7b4r1?file=/src/index.tsx

![image](https://user-images.githubusercontent.com/58159495/143260435-fbf9a552-7337-4ed5-84fa-d9e6ea0c5962.png)

Could you help me fix this, thanks.

**Comment by aaronkik** [comment]
I've just upgraded to the latest Chakra version and I'm not getting token suggestions too.

**Reply by ramesh-km** [reply]
Token suggestions working with version `@chakra-ui/react": "^1.6.12"`
